cli/src/main/java/dev/starfix/Starfix.java

```diff
@@ -100,7 +108,11 @@ public void run() {
             new CommandLine(new Starfix()).usage(System.out); // Will invoke Picocli Help
             return;
         }
+        try{
         cloneCmd(uri);
+        }catch(Exception e){
+            e.printStackTrace();
```

Comment:
never ok to just print out stacktrace in case of error. need to handle it or guide the user.